packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 type CustomStyleHook = (state: unknown) => void;
 
 // The list of hooks is built from the exports from react-components/src/index
-export type CustomStyleHooksContextValue = {
+export type CustomStyleHooksContextValue = Partial<{
```

Comment:
Checking the history of the hook, I see `Card` was added to the type in #27295 but the `Card*` components do not use the hooks?

Reply:
If that's the case it would be great to cover the custom styling hook presence by conformance tests.

Reply:
It was mentioned in the original PR that we should have some automation for this https://github.com/microsoft/fluentui/pull/26943#discussion_r1120015871

According to the discussion there should be a work item tracking this, could it be linked here?

Reply:
Oh yeah - https://github.com/microsoft/fluentui/issues/27676